packages/itwin/ec3-widget/src/components/ExportModal.tsx

```diff
@@ -0,0 +1,218 @@
+        try {
+          const jobRequest: EC3JobCreate = {
+            configurationId: props.templateId,
+            projectName: props.projectName,
+            ec3BearerToken: token,
+          };
+          const jobCreated = await ec3JobsClient.createJob(
+            accessToken,
+            jobRequest
+          );
+          if (jobCreated.id) {
+            pinStatus(jobCreated);
+          } else {
+            setJobStatus({ status: CarbonUploadState.Failed, message: "Failed to create Job" });
+          }
+        } catch (e) {
+          setJobStatus({ status: CarbonUploadState.Failed, message: "Missing required permissions. Please contact the project administrator." });
+          /* eslint-disable no-console */
+          console.error(e);
+        }
+      } else {
+        setJobStatus({ status: CarbonUploadState.Failed, message: "Invalid reportId" });
+      }
+    },
+    [props, pinStatus, ec3JobsClient, getAccessToken]
+  );
+
+  const onClose = useCallback(() => {
+    window.clearInterval(intervalRef.current);
+    props.close();
+  }, [props]);
+
+  const getStatusComponent = useCallback(
+    (state: JobStatus) => {
+      switch (state.status) {
+        case CarbonUploadState.Queued:
+          return (
+            <div className="ec3w-progress-radial-container">
+              <ProgressRadial indeterminate size="small" value={50} />
+              <Text variant="leading" className="ec3w-status-text">
+                Export queued
+              </Text>
+            </div>
+          );
+        case CarbonUploadState.Running:
+          return (
+            <div className="ec3w-progress-linear-container">
+              <ProgressLinear indeterminate />
+              <Text variant="leading" className="ec3w-status-text">
+                Export running
+              </Text>
+            </div>
+          );
+        case CarbonUploadState.Succeeded:
+          return (
+            <div className="ec3w-progress-radial-container">
+              <ProgressRadial status="positive" size="small" value={50} />
+              <a
+                className="ec3w-report-button"
+                href={state.link}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <Button styleType="cta">Open in EC3</Button>
+              </a>
+            </div>
+          );
+        case CarbonUploadState.Failed:
+          return (
+            <div className="ec3w-progress-radial-container">
+              <ProgressRadial status="negative" size="small" value={100} />
+              <Text variant="leading" className="ec3w-status-text">
+                Export failed <br />
+                {state.message}
+              </Text>
+            </div>
+          );
+        default:
+          return (
+            <div className="ec3w-progress-radial-container">
+              <Text>Invalid Job Status <span role="img" aria-label="sad">😔</span></Text>
```

Comment:
Why did you add the sad face? I do not think it is a good idea. This will make the product look less professional.

Reply:
Somewhere in the viewer I remember there being an emoji when an error occurred. Where do we draw the line between informal and rigidness.

Reply:
Was already here. Other widgets also have sad faces so it was probably copied. Deleted the sad face.

Reply:
Resolved